### PR TITLE
Fixed sonar issues found in RoleServiceImpl and AdapterComponentDocRe…

### PR DESCRIPTION
**Pull request by danielrf123**
…positoryOrchImpl

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImpl.java [hunk 1]
```diff
@@ -123,14 +126,14 @@ public UTCDateUtil getDateUtil() {
                 // Home Community
                 // ----------------
                 if ((homeCommunityId == null) && (oDocRequest.getHomeCommunityId() != null)
-                    && (oDocRequest.getHomeCommunityId().length() > 0)) {
+                        && (oDocRequest.getHomeCommunityId().length() > 0)) {
```
Comment (mhpnguyen) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImpl.java:129]: Optional: try to use StringUtil to check for null value

Comment (mhpnguyen) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImpl.java:129]: I've reviewed PR and endorse for merge without further comments

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/dao/UserLoginDAOImpl.java [hunk 2]
```diff
@@ -26,12 +26,8 @@
  */
 package gov.hhs.fha.nhinc.admingui.hibernate.dao;
 
-import gov.hhs.fha.nhinc.admingui.model.Login;
-import gov.hhs.fha.nhinc.admingui.services.exception.UserLoginException;
-import gov.hhs.fha.nhinc.admingui.services.persistence.jpa.entity.RolePreference;
-import gov.hhs.fha.nhinc.admingui.services.persistence.jpa.entity.UserLogin;
-import gov.hhs.fha.nhinc.admingui.services.persistence.jpa.entity.UserRole;
 import java.util.List;
+
```
Comment (vdmehta06) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/dao/UserLoginDAOImpl.java:30]: Remove empty line

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/dao/UserLoginDAO.java [hunk 3]
```diff
@@ -26,12 +26,13 @@
  */
 package gov.hhs.fha.nhinc.admingui.hibernate.dao;
 
+import java.util.List;
+
```
Comment (vdmehta06) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/dao/UserLoginDAO.java:30]: remove empty line in import

Comment (danielrf123) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/dao/UserLoginDAO.java:30]: My eclipse formatter is forcing that empty line to be inserted between the two group of imports.

**Pull request merged**